WalletWasabi.Packager/MacSignTools.cs

```diff
@@ -0,0 +1,394 @@
+				
+				var fileName = new FileInfo(file).Name;
+
+				if (fileName == ".DS_Store")
+				{
+					File.Delete(file);
+					continue;
+				}
+
+				var isExecutable = exacutableFileNames.Contains(fileName);
+
+				if (isExecutable)
+				{
+					using var process = Process.Start(new ProcessStartInfo
+					{
+						FileName = "chmod",
+						Arguments = $"u+x \"{file}\"",
+						WorkingDirectory = workingDir
+
+					});
+					process.WaitForExit();
+				}
+
+				var entitlementArgs = isExecutable ? entitlementsSandBoxPath : entitlementsPath;
+
+				Sign($"{signArguments} --entitlements \"{entitlementArgs}\" \"{file}\"", dmgPath);
+
+			}
+
+			Console.WriteLine("Phase: verifying the signature.");
+
+			Verify(appPath);
+
+			// Building a package - this is not required now. This code will be useful later - do not delete!
+			//var pkgPath = Path.Combine(workingDir, $"Wasabi-{versionPrefix}.pkg");
+			//using (var process = Process.Start(new ProcessStartInfo
+			//{
+			//	FileName = "productbuild",
+			//	Arguments = $"--component \"{appPath}\" /Applications \"{pkgPath}\" --sign \"L233B2JQ68\"",
+			//	WorkingDirectory = dmgPath
+			//}))
+			//{
+			//	process.WaitForExit();
+			//}
+
+			Console.WriteLine("Phase: notarize the app.");
+
+			ZipFile.CreateFromDirectory(appPath, appNotarizeFilePath);
+			Notarize(appleId, password, appNotarizeFilePath, bundleIdentifier);
+			Staple(appPath);
+
+			Console.WriteLine("Phase: creating the dmg.");
+
+			if (File.Exists(dmgFilePath))
+			{
+				File.Delete(dmgFilePath);
+			}
+
+			var dmgArguments = string.Join(" ", new string[]
+			{
+				"--volname \"Wallet Wasabi\"",
+				$"--volicon \"{Path.Combine(resPath, "WasabiLogo.icns")}\"",
+				$"--background \"{Path.Combine(contentsPath, "Logo_with_text_small.png")}\"",
+				"--window-pos 200 120",
+				"--window-size 600 450",
+				"--icon-size 100",
+				"--icon \"Wasabi Wallet.app\" 100 150",
+				"--hide-extension \"Wasabi Wallet.app\"",
+				"--app-drop-link 500 150",
+				"--no-internet-enable",
+				$"\"{dmgFilePath}\"",
+				$"\"{appPath}\""
+			});
+
+			Console.WriteLine("Phase: creating dmg.");
+
+			using (var process = Process.Start(new ProcessStartInfo
+			{
+				FileName = "create-dmg",
+				Arguments = dmgArguments,
+				WorkingDirectory = dmgPath
+			}))
+			{
+				process.WaitForExit();
+			}
+
+			Console.WriteLine("Phase: signing the dmg file.");
+
+			UnlockKeychain();
+
+			Sign($"{signArguments} --entitlements \"{entitlementsPath}\" \"{dmgFilePath}\"", dmgPath);
+
+			Console.WriteLine("Phase: verifying the signature.");
+
+			Verify(dmgFilePath);
+
+			Console.WriteLine("Phase: notarize dmg");
+			Notarize(appleId, password, dmgFilePath, bundleIdentifier);
+
+			Console.WriteLine("Phase: staple dmp");
+			Staple(dmgFilePath);
+
+			File.Move(dmgFilePath, desktopDmgFilePath);
+			DeleteWithChmod(workingDir);
+			File.Delete(zipPath);
+
+			Console.WriteLine("Phase: finish.");
+		}
+
+		public static bool IsMacSignMode(string[] args)
```

Comment:
It is not using the parameter.
```suggestion
		public static bool IsMacSignMode()
```